rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java

```diff
@@ -99,19 +101,40 @@ public static void main(String[] args) throws Exception {
     public static class EnrichmentFunction
```

Comment:
Since class implements `Serializable` interface I recommend to add `serialVersionUID`